frontend/src/main-components/Sidebar.js

```diff
@@ -1,48 +1,47 @@
 import React from "react";
 import styled from "@emotion/styled";
-
-import accountIcon from "../SVG/account.svg";
 import logoutIcon from "../SVG/login-black-18dp.svg";
-import newCardIcon from "../SVG/create-24px.svg"
+import newCardIcon from "../SVG/create-24px.svg";
 import objectIcon from "../SVG/image_search-24px.svg";
-import youTubeIcon from "../SVG/youtube.svg";
-import newFolderIcon from "../SVG/create_new_folder-24px.svg"
+import gameIcon from "../SVG/game.svg";
+import newFolderIcon from "../SVG/create_new_folder-24px.svg";
 import homeIcon from "../SVG/home-24px.svg";
 import SideIcon from "../sub-components/SideIcon";
 
 const Sidebar = (props) => {
   let sideStyling = "width: 0; border: none;";
-  if (props.bool === "t") {
-     sideStyling = "width: 20%; border: .3rem solid black; align-items:flex-start;"
+  if (props.visible) {
+    sideStyling =
+      "width: 20%; border: .3rem solid black; align-items:flex-start;";
   }
-  
-   const Side = styled.div`
-     flex: 1;
-     max-width: 400px;
-     display: flex;
-     flex-direction: column;
-     align-items: center;
-     background-color: white;
-     border: 1rem solid white;
-     border-radius: 1rem;
-     margin: 1%;
-     @media (max-width: 700px) {
-       ${sideStyling}
-       overflow-x: hidden;
-       position: fixed;
-     }
-   `;
+
+  const Side = styled.div`
```

Comment:
Outside.

Reply:
done.

Reply:
This actually doesn't work because the styled div contains a string that becomes undefined if I pull it out. This string changes the css code that makes the sidebar visible.

Reply:
done.